sass/cd/cd-footer/_cd-footer-menu.scss

```diff
@@ -9,42 +9,45 @@
   margin: 0;
   padding-left: 0;
   text-transform: uppercase;
+
+  @include tablet {
+    flex: 1 0 100%;
+  }
+  @include desktop {
+    text-align: left;
+    flex: 1 0 70%;
+  }
 }
 
-.cd-footer-menu > li,
 .cd-footer-menu__item {
-  margin-bottom: 20px;
+  display: block;
+  margin-bottom: 12px;
 
-  &:last-child {
-    margin-bottom: 0;
+  @include tablet {
+    display: inline-block;
+    margin-right: 16px;
```

Comment:
If these items are centered, I think the margin-right throws off their center alignment

Reply:
Good catch! This isn't a new style, I was just reorganizing and didn't consider whether stuff should or shouldn't be there. The removal is here: https://github.com/UN-OCHA/ocha_basic/pull/31/files#diff-65d963d13dc0dc9744bdf5591a56e5d7L31

New commit splits the margin to L/R on tablet but retains right-only on desktop to ensure the nav sits flush with left edge when layout changes.

![screenshot 2018-08-21 15 14 20](https://user-images.githubusercontent.com/254753/44403602-341e3c00-a555-11e8-984b-7fde8c7d7ef1.png)